### PR TITLE
zh-translation: /docs/root/configuration/http/http_filters/tap_filter.rst

### PR DESCRIPTION
**Pull request by bjtuchenmin**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: /docs/root/configuration/http/http_filters/tap_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #257
[Optional Deprecated:]

### CODE REVIEW

**Comment by addozhang** [comment]
@bjtuchenmin 编译出错了

File: docs/root/configuration/http/http_filters/tap_filter.rst [hunk 1]
```diff
@@ -37,27 +30,20 @@ Example filter configuration:
       admin_config:
         config_id: test_config_id
 
-The previous snippet configures the filter for control via the :http:post:`/tap` admin handler.
-See the following section for more details.
-
-.. _config_http_filters_tap_admin_handler:
+上面的代码段通过 :http:post:`/tap` admin handler 配置了过滤器以便管理。详情请参考下一节。
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/tap_filter.rst]:
```suggestion
上面的代码段通过 :http:post:`/tap` admin handler 配置了过滤器以便管理。详情请参考下一节。

.. _config_http_filters_tap_admin_handler:
```

File: docs/root/configuration/http/http_filters/tap_filter.rst [hunk 2]
```diff
@@ -4,29 +4,22 @@ Tap
 ===
 
 * :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.tap.v3.Tap>`
```
Comment (majinghe) [inline docs/root/configuration/http/http_filters/tap_filter.rst]:
```suggestion
* :ref:`v3 API 参考 <envoy_v3_api_msg_extensions.filters.http.tap.v3.Tap>`
```

File: docs/root/configuration/http/http_filters/tap_filter.rst [hunk 3]
```diff
@@ -37,27 +30,22 @@ Example filter configuration:
       admin_config:
         config_id: test_config_id
 
-The previous snippet configures the filter for control via the :http:post:`/tap` admin handler.
-See the following section for more details.
+上面的代码段通过 :http:post:`/tap` admin handler 配置了过滤器以便管理。详情请参考下一节。
 
 .. _config_http_filters_tap_admin_handler:
 
 Admin handler
```
Comment (majinghe) [inline docs/root/configuration/http/http_filters/tap_filter.rst]:
```suggestion
管理处理器（admin handler）
```
结合下文，这个应该可以如上翻译，后文统一用管理处理器。

File: docs/root/configuration/http/http_filters/tap_filter.rst [hunk 4]
```diff
@@ -37,27 +30,22 @@ Example filter configuration:
...
 -------------
 
-When the HTTP filter specifies an :ref:`admin_config
-<envoy_v3_api_msg_extensions.common.tap.v3.AdminConfig>`, it is configured for admin control and
-the :http:post:`/tap` admin handler will be installed. The admin handler can be used for live
-tapping and debugging of HTTP traffic. It works as follows:
+当 HTTP 过滤器指定了一个 :ref:`admin_config
+<envoy_v3_api_msg_extensions.common.tap.v3.AdminConfig>`, 它将被配置用来管理员控制，且 :http:post:`/tap` admin handler 将被安装。 admin handler 可以用来实时过滤和调试 HTTP 流量。工作流程如下：
```
Comment (majinghe) [inline docs/root/configuration/http/http_filters/tap_filter.rst]:
```suggestion
<envoy_v3_api_msg_extensions.common.tap.v3.AdminConfig>`, 它将被配置用来管理员控制，且 :http:post:`/tap` 管理处理器将被安装。 管理处理器可以用来实时过滤和调试 HTTP 流量。工作流程如下：
```

**Comment by majinghe** [comment]
@bjtuchenmin 很棒的翻译，一点小建议。

**Review by addozhang — APPROVED** [review]
/LGTM

File: docs/root/configuration/http/http_filters/tap_filter.rst [hunk 5]
```diff
@@ -37,27 +30,22 @@ Example filter configuration:
...
 
-The previous snippet configures the filter for control via the :http:post:`/tap` admin handler.
-See the following section for more details.
+上面的代码段通过 :http:post:`/tap` admin handler 配置了过滤器以便管理。详情请参考下一节。
 
 .. _config_http_filters_tap_admin_handler:
 
-Admin handler
+管理处理器（admin handler）
 -------------
```
Comment (addozhang) [inline docs/root/configuration/http/http_filters/tap_filter.rst]:
```suggestion
--------------------------
```

**Review by addozhang — APPROVED** [review]
/LGTM

**Pull request merged**